src/main/groovy/com/cloudogu/smp/Dependencies.groovy

```diff
@@ -65,13 +65,21 @@ class Dependencies {
     // but remove the core dependencies before packaging
     // https://github.com/gradle/gradle/blob/master/subprojects/plugins/src/main/java/org/gradle/api/plugins/WarPlugin.java#L72
     ConfigurationContainer configurationContainer = project.getConfigurations()
+
+    def resolutionStrategy = {
```

Comment:
The name tells, what it is, but not what it does. Would like something like `noCacheResolutionStrategy` more.